pkg/agent/openflow/pipeline.go

```diff
@@ -274,6 +271,10 @@ var (
 	// metricEgressRuleIDRange takes 32..63 range of ct_label to store the egress rule ID.
 	metricEgressRuleIDRange = binding.Range{32, 63}
 
+	// traceflowMarkToSRange stores dataplaneTag at range 2-7 in ToS field of IP header.
+	// IPv4/v6 DSCP (bits 2-7) field supports exact match only.
+	traceflowMarkToSRange = binding.Range{2, 7}
```

Comment:
why "Mark" in the name and not "Tag"?

Reply:
Because this is used to "Mark" a packet as Traceflow packet, but I'm OK to change this to "Tag" if we think "Tag" is better.

Reply:
I think it would be better, since we typically use "Mark" for 1-bit flags

Reply:
Got it. Changed this from "Mark" to "Tag".